README.md

```diff
@@ -1,56 +1,35 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
-
+# Hello world Project
+> This project display hello word.
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
+- HTML5
+- CSS3
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
-
 ### Prerequisites
-
+#### You need to have a git installed
+#### You need to have a basic knowledge about HTML and CSS
 ### Setup
 
+#### clone  this project (using git clone https://github.com/bizip/hello-world.git) to your local computer and use telminal to navigate into the parent directory of this project.
 ### Install
 
+#### Run npm install
 ### Usage
```

Comment:
- [ ] OPTIONAL: Good job on adding getting started section in README file, but because you are using headings instead of paragraphs, the readme has become difficult to read. In case you don't know yet, `###` and `##` are both usually headings of different sizes. I recommend you to delete `###` from paragraphs that are describing sections. You can try this: 
```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/bizip/hello-world.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/bizip/hello-world.git`.
- go to the repository folder in your command prompt `cd Hello-Page-Setup`

```